### PR TITLE
MNTOR-2326 - Update footer to show terms of service & privacy notice

### PR DESCRIPTION
**Pull request by codemist**
# References: 
Jira: MNTOR-2326

# Screenshot (if applicable)

image

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

**Review by flodolo — APPROVED** [review]

File: src/app/(nextjs_migration)/(authenticated)/user/layout.tsx [hunk 1]
```diff
@@ -141,10 +141,18 @@ const MainLayout = async (props: Props) => {
           </li>
           <li>
             <a
-              href="https://www.mozilla.org/privacy/firefox-monitor"
+              href="https://www.mozilla.org/en-US/about/legal/terms/subscription-services/"
```
Comment (pdehaan) [inline src/app/(nextjs_migration)/(authenticated)/user/layout.tsx:144]: nit: remove language specific path and let Bedrock handle best language.
```suggestion
              href="https://www.mozilla.org/about/legal/terms/subscription-services/"
```

File: src/app/(nextjs_migration)/(authenticated)/user/layout.tsx [hunk 2]
```diff
@@ -141,10 +141,18 @@ const MainLayout = async (props: Props) => {
...
               target="_blank"
             >
-              {l10n.getString("terms-and-privacy")}
+              {l10n.getString("terms-of-service")}
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.mozilla.org/en-US/privacy/subscription-services/"
```
Comment (pdehaan) [inline src/app/(nextjs_migration)/(authenticated)/user/layout.tsx:152]:
```suggestion
              href="https://www.mozilla.org/privacy/subscription-services/"
```

File: src/app/(nextjs_migration)/(guest)/layout.tsx [hunk 3]
```diff
@@ -116,10 +116,18 @@ const GuestLayout = async (props: Props) => {
           </li>
           <li>
             <a
-              href="https://www.mozilla.org/privacy/firefox-monitor"
+              href="https://www.mozilla.org/en-US/about/legal/terms/subscription-services/"
```
Comment (pdehaan) [inline src/app/(nextjs_migration)/(guest)/layout.tsx:119]:
```suggestion
              href="https://www.mozilla.org/about/legal/terms/subscription-services/"
```

File: src/app/(proper_react)/redesign/Shell.tsx [hunk 4]
```diff
@@ -96,8 +96,19 @@ export const Shell = (props: Props) => {
...
+                  <a
+                    href="https://www.mozilla.org/en-US/about/legal/terms/subscription-services/"
+                    target="_blank"
+                  >
+                    {l10n.getString("terms-of-service")}
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href="https://www.mozilla.org/en-US/privacy/subscription-services/"
```
Comment (pdehaan) [inline src/app/(proper_react)/redesign/Shell.tsx:108]:
```suggestion
                    href="https://www.mozilla.org/privacy/subscription-services/"
```

**Review by Vinnl — APPROVED** [review]

**Pull request merged**